<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vendorPopup</class>
 <widget class="QDialog" name="vendorPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Vendor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="vendor_entry_form_frame">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="9">
       <widget class="QLabel" name="vendor_comission_percent">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="vendor_customs_label">
        <property name="text">
         <string>Customs</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="target_profit_percent_label_2">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="vendor_shippingcost_label">
        <property name="text">
         <string>Shipping Cost</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="vendor_currency_label">
        <property name="text">
         <string>Vendor Currency:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="vendor_leadtime_label">
        <property name="text">
         <string>Vendor Lead Time (Weeks):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="target_profit_label_2">
        <property name="text">
         <string>Vendor Commission:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="vendor_lead_time_field"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="vendor_packingcost_label">
        <property name="text">
         <string>Packing &amp; Legalization Cost:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5" colspan="4">
       <widget class="QLineEdit" name="vendor_comission_field">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="packing_cost_field"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="customs_percent_field"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="vendor_name_label">
        <property name="text">
         <string>Vendor Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="10">
       <widget class="QLineEdit" name="vendor_name_field"/>
      </item>
      <item row="2" column="5" colspan="6">
       <widget class="QLineEdit" name="shipping_cost_field">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="5" colspan="2">
       <widget class="QComboBox" name="vendor_currency_combo">
        <item>
         <property name="text">
          <string>EUR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GBP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EGP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AED</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>135</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="vendorPopupBtnBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>vendorPopupBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>vendorPopup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vendorPopupBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>vendorPopup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
